docs/reference/scripting/painless.asciidoc

```diff
@@ -1,32 +1,35 @@
 [[modules-scripting-painless]]
 == Painless scripting language
 
-_Painless_ is a simple, secure scripting language designed specifically for use
-with Elasticsearch. It is the default scripting language for Elasticsearch and
-can safely be used for inline and stored scripts. To get started with
-Painless, see the {painless}/painless-guide.html[Painless Guide]. For a
-detailed description of the Painless syntax and language features, see the
-{painless}/painless-lang-spec.html[Painless Language Specification].
+_Painless_ is a performant, secure scripting language designed specifically for
+{es}. You can use Painless anywhere scripts are supported in {es} to safely
+write inline and stored scripts.
 
 [[painless-features]]
-You can use Painless anywhere scripts can be used in Elasticsearch. Painless
-provides:
-
-* Fast performance: Painless scripts https://benchmarks.elastic.co/index.html#search_qps_scripts[
-run several times faster] than the alternatives.
-
-* Safety: Fine-grained allowlists with method call/field granularity. See the
-{painless}/painless-api-reference.html[Painless API Reference] for a
-complete list of available classes and methods.
-
-* Optional typing: Variables and parameters can use explicit types or the
-dynamic `def` type.
-
-* Syntax: Extends a subset of Java's syntax to provide additional scripting
-language features.
-
-* Optimizations: Designed specifically for Elasticsearch scripting.
-
-Ready to start scripting with Painless? See the
-{painless}/painless-guide.html[Painless Guide] for the
-{painless}/index.html[Painless Scripting Language].
\ No newline at end of file
+Painless provides numerous capabilities that center around the following
+core principles:
+
+* **Safety**: Ensuring the security of your cluster is of utmost importance. To
+that end, Painless uses a fine-grained allowlist with a granularity down to the
+members of a class. Anything that is not part of the allowlist simply does not
+execute. See the {painless}/painless-api-reference.html[Painless API Reference]
+for a complete list of available classes, methods, and fields per script
+context.
+* **Performance**: Painless compiles directly into JVM bytecode to take
+advantage of all possible optimizations the JVM provides. Also, Painless
+typically avoids features that require additional slower checks at runtime.
+* **Simplicity**: Painless implements a syntax with a natural familiarity to
+anyone with some basic coding experience. Painless uses a subset of Java syntax
+with some additional improvements to enhance readability and remove
```

Comment:
Highlighting `def` here would make this point solid.  Right now it's pretty mushy.

Reply:
Does `def` extend a basic Java class? How do we want to solidify this point? Something like:
>For example, the dynamic type (`def`) builds on the XYZ Java class to mimic other types and simplify how you write scripts.

Reply:
I don't have a good one liner that fits here.  Withdrawn.